assets/src/edit-story/app/rightClickMenu/provider.js

```diff
@@ -147,7 +169,63 @@ function RightClickMenuProvider({ children }) {
     deleteCurrentPage();
   }, [deleteCurrentPage]);
 
-  const selectedElement = selectedElements?.[0];
+  /**
+   * Send element one layer backwards, if possible.
+   */
+  const handleSendBackward = useCallback(() => {
+    const newPosition =
+      currentPosition === 1 ? currentPosition : currentPosition - 1;
+
+    arrangeElement({
+      elementId: selectedElement.id,
+      position: newPosition,
+    });
```

Comment:
I don't think this should ever be the case with the check above, but might be safer to clamp this. idk. I'm cool if you think it's unnecessary too

Reply:
Chatted with max but posting here for visibility: line 177 clamps this so it doesn't go below position 1 - the first layer